src/main/java/org/teacon/slides/render/ProjectorWorldRender.java

```diff
@@ -0,0 +1,94 @@
+package org.teacon.slides.render;
+
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.IVertexBuilder;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.player.ClientPlayerEntity;
+import net.minecraft.client.renderer.IRenderTypeBuffer;
+import net.minecraft.client.renderer.Matrix4f;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.World;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.RenderWorldLastEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.teacon.slides.ProjectorTileEntity;
+
+//Source: https://github.com/McJty/YouTubeModding14/blob/master/src/main/java/com/mcjty/mytutorial/client/InWorldRenderer.java
+@Mod.EventBusSubscriber(Dist.CLIENT)
+public class ProjectorWorldRender {
+    @SubscribeEvent
+    public static void worldRender(final RenderWorldLastEvent event) {
+        ClientPlayerEntity player = Minecraft.getInstance().player;
+        if (player.getHeldItemMainhand().getItem() == Items.STICK) {
```

Comment:
I'm not sure if identifying a stick is a good choice ... holding a projector seems more "natural" since players can easily discover this, but holding a stick will not be immediately thought about.